### PR TITLE
[DSD-4658] updated the opencrvs installation scripts

### PR DESCRIPTION
**Pull request by Rakshitha650**

### CODE REVIEW

File: deployment/install.sh [hunk 1]
```diff
@@ -17,12 +17,61 @@ kubectl label ns $NS istio-injection=enabled --overwrite
...
 
+mosip_opencrvs_partner_client_id="opencrvs-partner"
+mosip_opencrvs_partner_client_secret="qYkjsUASf3cwakJOSXvYL38U8o3pGU6g"
+mosip_opencrvs_partner_client_sha_secret="b8247112-f810-4848-a854-ce0699ee16b9"
+mosip_opencrvs_uin_token_partner="opencrvs-auth-partner"
+
+opencrvs_client_id="89a383a4-57fe-4392-8247-97e05441741b"
+opencrvs_client_secret_key="8b5ad79e-a33a-4bad-810e-3865f886e9d6"
+opencrvs_client_sha_secret="b8247112-f810-4848-a854-ce0699ee16b9"
+
```
Comment (syedsalman3753) [inline deployment/install.sh]: please take the input from the user and avoid hardcoding of any values

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: deployment/install.sh (hunk 1 above)
Comment (Rakshitha650) [reply deployment/install.sh]: done @syedsalman3753

Comment (Rakshitha650) [reply deployment/install.sh]: done @syedsalman3753

**Review by syedsalman3753 — APPROVED** [review]

**Review by ckm007 — COMMENTED** [review]
Why client secret is hardcoded ?

**Review by ckm007 — APPROVED** [review]

**Pull request merged**